test/e2e/network_segmentation.go

```diff
@@ -486,7 +490,7 @@ var _ = Describe("Network Segmentation", func() {
 						*podConfig(
 							"udn-pod",
 							withCommand(func() []string {
-								return httpServerContainerCmd(port)
+								return httpServerContainerCmd(podClusterNetPort)
```

Comment:
there are a handful more of these changes that should have been part of an earlier commit

Reply:
should be fixed.